frontend/src/views/Game.tsx

```diff
@@ -199,20 +201,69 @@ function GamePage() {
     }
   };
 
+  // Send notification if solution is correct and currentUser is set.
+  const checkSendSolutionCorrectNotification = (submissionParam: Submission) => {
```

Comment:
If the other notification method is no longer being used could we delete it?

Reply:
Oh I just saw it's still being called in the other runSolution method - does this mean that if someone checks against a custom test case, other people will see that they tested it locally?

Reply:
Yeah that's right - I'm thinking that all of these notifications could be incorporated into an "activity feed" similar to [binarysearch](https://binarysearch.com/)'s chat functionality, but just in a separate feed (or perhaps a toggle-able feed).